src/content/components/TreeView.js

```diff
@@ -306,6 +309,14 @@ class TreeView extends Component {
   }
 
   _onKeyDown(event) {
+    if ((event.ctrlKey || event.metaKey) && event.keyCode === 67) { // COPY
```

Comment:
I'm not a fan of manual key handling for this. Please use the copy event for this. See the code example at https://developer.mozilla.org/en-US/docs/Web/Events/copy .

In order to use this, you'll need to:
 1. Remove user-select:none from .treeView and instead put it on .treeViewRow and .treeViewHeader
 2. Add an onCopy attribute + prop to the VirtualList container element
 3. Handle this onCopy callback in the TreeView component